src/pydata_sphinx_theme/assets/styles/components/_searchbox.scss

```diff
@@ -24,8 +24,7 @@ div#searchbox {
       font-size: 1.25rem;
       padding: 0.75rem;
       background-color: var(--pst-color-primary);
-      // Button text is always white with Sphinx Design buttons
-      color: white;
+      color: var(--pst-color-primary-text);
```

Comment:
Thanks for flagging this! Honestly this is something I completely missed as I created some of these variables on the fly.

> where the intended page element (either -bg or -text) comes last.
This makes sense, will make a pass to fix these

Reply:
On a quick check some variables like `text-base` seem to have already been there when I made the change (see in [colour.scss](https://github.com/trallard/pydata-sphinx-theme/blob/e5830a28fb760cc15688fcfa4dc4e8a4600225d7/src/pydata_sphinx_theme/assets/styles/variables/_color.scss#L168)) but I can go ahead and unify these

Reply:
I just noticed that since [sphinx_design.scss creates its variables to work with sphinx design components](https://github.com/trallard/pydata-sphinx-theme/blob/559b1f789df45b1ce7783f3d02e64840ce3d14b8/src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss#L71) this effectively overrides what would then be `--pst-color-muted-text` which now makes me believe this is the reason why  `--pst-color-text-muted` was defined in the first place